xbmc/addons/kodi-addon-dev-kit/doxygen/General/DoxygenOnAddon.dox

```diff
@@ -23,52 +23,6 @@ Start doxygen documentation for add-ons always with `///` and on Kodi itself wit
 <b>Here an example on add-on about function coding style:</b>
 
 \verbatim
-#ifdef DOXYGEN_SHOULD_USE_THIS
```

Comment:
either add a different example here or remove the lines above too, which mention that an example will follow

Reply:
done